doc/project/release_process.rst

```diff
@@ -94,8 +94,15 @@ At that point, the whole process starts over again.
 Release Quality Criteria
 ************************
 
-The current backlog of prioritized bugs shall be used as a quality metric to
-gate the final release. The following counts shall be used:
+Static Analysis results are a fundamental requirement for the
+project. The final release must not contain any high-critical issues
+that can potentially compromise the functionality, security, or
+reliability of our software.  High-critical issues represent
+vulnerabilities that, if left unresolved, could have severe
+consequences.
```

Comment:
Do I understand correctly that this high-critical issues are marked with the "priority: high" Tag? Or is this orthogonal to the low/medium/high priority tagging?

Reply:
yep, you understood correctly. high critical issues are tagged with high priority